docs/merged_reduced_scans_table.md

```diff
@@ -101,13 +101,10 @@ Mismatch Errors are used when the connection is successful, but the content rece
 | http.empty              | Received no content when HTTP content was expected |
 | http.truncated_response | The HTTP response content was unexpectedly truncated |
 |                         |
-| **Mismatch Errors**     | The connection completed successfully, but the content returned didn't match the content expected for the domain. |
+| **Unexpected Content**  | The connection completed successfully, but the content returned didn't match the content expected for the domain. |
 |                         |
-| HyperQuack v1 only      |
-| response_mismatch       | Received a different response from the one expected. </br> For Discard no response is expected and any response is a mismatch, </br> for Echo a mirrored response is expected and anything else is a mismatch. |
-| status_mismatch         | The HTTP status code didn't match, eg. `403` instead of `200` |
-| body_mismatch           | The HTTP body didn't match, potentially a blockpage |
-| tls_mismatch            | An element of the TLS connection (certificate, cipher suite, or TLS version) didn't match |
-| HyperQuack v1 and v2    |
-| template_mismatch       | Some element of the response did not match the expected template |
+| unexpected              | Received a different response from the one expected. </br> For Discard no response is expected and any response is a mismatch, </br> for Echo a mirrored response is expected and anything else is a mismatch. </br> For HTTP/S the expected response is determined by sending multiple control domains to the server and building an expected template. This response is returned when more detail about the exact part of the template mismatched (eg. status, body) is not available. |
+| unexpected_status       | The HTTP status code didn't match, eg. `403` instead of `200` |
```

Comment:
I don't think the new naming makes things clearer.

`unexpected_X` doesn't explain why `X` is unexpected. If I get status 404, is that an unexpected status?

I think the `X_mismatch` is better, because the `_mismatch` makes it clearer that you are comparing against something, the control.

Furthermore, it places the important thing, `X`, on the left, which can mitigate truncation issues in the legends.

Reply:
I see that all you are doing is renaming some outcomes and removing `template_mismatch`.
I think it's fine to still remove `template_mismatch`, but I'd replace it with the generic `response_ mismatch`, and keep the other names.

Reply:
Good point about the truncation.

I switched them back to `content/*_mismatch`, but turned `content/template_mismatch` and `content/response_mismatch` into just `content/mismatch`.

I think that helps accomplish my goal of making it clearer that `content/*_mismatch` outcomes are special cases of `content/mismatch` outcomes. 

If you still prefer `content/response_mismatch` let me know and I'll change it back.

Reply:
I think that's fine.